app/Models/EnvironmentVariable.php

```
@@ -349,7 +371,7 @@ private function set_environment_variables(?string $environment_variable = null)
     protected function key(): Attribute
     {
         return Attribute::make(
-            set: fn (string $value) => str($value)->trim()->replace(' ', '_')->value,
+            set: fn (string $value) => ValidationPatterns::validatedEnvironmentVariableKey($value),
         );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Database\Eloquent\Casts\Attribute' in method 'key'. (undefined)

(StaticAccess)